### ISSUE TITLE
Unable to load Redis extension with PHP 7.4

### ISSUE THREAD

**Issue by linuxjuggler** [shivammathur/setup-php#96]
**Describe the bug**
Loading redis extension with PHP 7.4 produce the following error message

> PHP Warning:  PHP Startup: Unable to load dynamic library 'redis.so' (tried: /usr/lib/php/20190902/redis.so (/usr/lib/php/20190902/redis.so: undefined symbol: igbinary_serialize), /usr/lib/php/20190902/redis.so.so (/usr/lib/php/20190902/redis.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

**To Reproduce**
```yml
      - name: Install PHP
        uses: shivammathur/setup-php@master
        with:
          php-version: 7.4
          extension-csv: redis
```

or 
```yml
      - name: Install PHP
        uses: shivammathur/setup-php@master
        with:
          php-version: 7.4
          extension-csv: redis
          pecl: true
```

Both will cause the same issue

**Expected behavior**
Being able to load redis without any problem

**Screenshots**
![Screen Shot 2019-12-02 at 5 24 14 PM](https://user-images.githubusercontent.com/27624/69967082-bf43fb00-1528-11ea-94b7-7b48ebb76cd1.png)

**Additional context**

**Comment by shivammathur** [comment]
@linuxjuggler Thanks for reporting.
Fixed in #101, [Test workflow](https://github.com/shivammathur/test-setup-php/commit/cc735ef8892aa8b481eb5a32e83767620e7c4fec/checks?check_suite_id=338223014)

This should work now, on all OS.
```yml
- name: Install PHP
  uses: shivammathur/setup-php@v1
  with:
    php-version: 7.4
    extension-csv: redis
```

**Comment by linuxjuggler** [comment]
Amazing, thanks @shivammathur
